<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open Vet</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/icone.png</normaloff>../images/icone.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_fiches">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>1076</width>
      <height>171</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab_fiche1">
     <attribute name="title">
      <string>Client1</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="LayoutFicheHead">
        <item>
         <widget class="QFrame" name="frame_client">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>120</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>520</width>
            <height>120</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(188, 255, 231,0);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="MyComboBox" name="comboBox_client">
                 <property name="minimumSize">
                  <size>
                   <width>350</width>
                   <height>27</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>500</width>
                   <height>27</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>Sélection Client</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">selection-background-color: rgb(255, 255, 127);</string>
                 </property>
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_editClient">
                 <property name="maximumSize">
                  <size>
                   <width>26</width>
                   <height>26</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>Modifie client</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../images/edit1.png</normaloff>../images/edit1.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>10</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_ClientInfo">
                 <property name="maximumSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>Informations complémentaires</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/newPrefix/images/info.png</normaloff>:/newPrefix/images/info.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_scan1">
                 <property name="maximumSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>Recherche client / animal</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../images/loupe2.png</normaloff>../images/loupe2.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTextBrowser" name="textBrowser_client">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Double click pour sélectionner l'édition</string>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(231, 231, 231);</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::Box</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="lineWidth">
                <number>1</number>
               </property>
               <property name="midLineWidth">
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_animal">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>120</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>520</width>
            <height>120</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(188, 255, 231,0);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="MyComboBox" name="comboBox_Animal">
                 <property name="minimumSize">
                  <size>
                   <width>320</width>
                   <height>27</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>500</width>
                   <height>27</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>Selection Animal</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">selection-background-color: rgb(255, 255, 127);</string>
                 </property>
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_editAnimal">
                 <property name="maximumSize">
                  <size>
                   <width>26</width>
                   <height>26</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>Modifie animal</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../images/edit1.png</normaloff>../images/edit1.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_Poids">
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>poids et mesures</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/newPrefix/images/mesures.png</normaloff>:/newPrefix/images/mesures.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>36</width>
                   <height>36</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_Relance">
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/newPrefix/images/relance.png</normaloff>:/newPrefix/images/relance.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>36</width>
                   <height>36</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_scan2">
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>Recherche client / animal</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../images/loupe2.png</normaloff>../images/loupe2.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTextBrowser" name="textBrowser_animal">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string/>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(231, 231, 231);</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::Box</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_fiche2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Client2</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_new">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1088</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDossiers_M_dicaux">
    <property name="title">
     <string>Dossiers Médicaux</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuComptabilit">
    <property name="title">
     <string>Comptabilité</string>
    </property>
    <widget class="QMenu" name="menuD_clarations">
     <property name="title">
      <string>Déclarations</string>
     </property>
     <addaction name="actionTVA"/>
     <addaction name="action2035"/>
    </widget>
    <addaction name="action_Recettes"/>
    <addaction name="actionD_penses"/>
    <addaction name="action_Banque"/>
    <addaction name="actionRecoupement_bancaire"/>
    <addaction name="menuD_clarations"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Paramètres</string>
    </property>
    <widget class="QMenu" name="menuBase_de_donn_es">
     <property name="title">
      <string>Base de données</string>
     </property>
     <addaction name="actionBackup"/>
     <addaction name="actionRestore"/>
     <addaction name="actionImport"/>
    </widget>
    <addaction name="actionBureau"/>
    <addaction name="action_Facturation"/>
    <addaction name="action_M_dicaments"/>
    <addaction name="action_Pathologies"/>
    <addaction name="separator"/>
    <addaction name="menuBase_de_donn_es"/>
    <addaction name="actionR_seau_S_curit"/>
   </widget>
   <widget class="QMenu" name="menuQuitter">
    <property name="title">
     <string>Quitter</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuStock">
    <property name="title">
     <string>Stock</string>
    </property>
    <addaction name="actionCommandes"/>
    <addaction name="actionAnalyse"/>
   </widget>
   <widget class="QMenu" name="menuStatistiques">
    <property name="title">
     <string>Statistiques</string>
    </property>
   </widget>
   <addaction name="menuDossiers_M_dicaux"/>
   <addaction name="menuStock"/>
   <addaction name="menuComptabilit"/>
   <addaction name="menuStatistiques"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuQuitter"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Recettes">
   <property name="text">
    <string>&amp;Recettes</string>
   </property>
  </action>
  <action name="actionD_penses">
   <property name="text">
    <string>&amp;Dépenses</string>
   </property>
  </action>
  <action name="action_Banque">
   <property name="text">
    <string>&amp;Banque</string>
   </property>
  </action>
  <action name="actionRecoupement_bancaire">
   <property name="text">
    <string>Recoupement bancaire</string>
   </property>
  </action>
  <action name="actionTVA">
   <property name="text">
    <string>TVA</string>
   </property>
  </action>
  <action name="action2035">
   <property name="text">
    <string>2035</string>
   </property>
  </action>
  <action name="actionCommandes">
   <property name="text">
    <string>&amp;Commandes</string>
   </property>
  </action>
  <action name="actionAnalyse">
   <property name="text">
    <string>Visualisation</string>
   </property>
  </action>
  <action name="action_Facturation">
   <property name="text">
    <string>&amp;Facturation</string>
   </property>
  </action>
  <action name="actionBureau">
   <property name="text">
    <string>Bureau</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+B</string>
   </property>
  </action>
  <action name="action_M_dicaments">
   <property name="text">
    <string>&amp;Médicaments</string>
   </property>
  </action>
  <action name="action_Pathologies">
   <property name="text">
    <string>&amp;Pathologies</string>
   </property>
  </action>
  <action name="actionBackup">
   <property name="text">
    <string>Backup</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="text">
    <string>Restore</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionR_seau_S_curit">
   <property name="text">
    <string>Réseau/Sécurité</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="toolTip">
    <string>Quitter l'application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>Mywidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
